src/Stratis.Bitcoin.Features.LightWallet.Tests/LightWalletSyncManagerTest.cs

```diff
@@ -311,7 +311,7 @@ public void Stop_DisposesDependencies()
             var asyncLoop = new Mock<IAsyncLoop>();
             this.asyncLoopFactory.Setup(
                 a => a.RunUntil(
-                    "WalletFeature.DownloadChain",
+                    "LightWalletSyncManager.SyncFromHeight",
                     It.IsAny<CancellationToken>(),
```

Comment:
Actually cant we use:

```nameof(LightWalletSyncManager.SyncFromDate)```

?

Reply:
yeah we could, although I'm not sure it'll provide great value here.
Maybe something we can look at changing in all the async loops everywhere.

Reply:
OK cool! Ready to merge then.